src/crystal/system/win32/event_loop_iocp.cr

```diff
@@ -1,8 +1,36 @@
+require "c/ioapiset"
 require "crystal/system/print_error"
 
 module Crystal::EventLoop
   @@queue = Deque(Fiber).new
 
+  # Returns the base IO Completion Port
+  class_getter iocp : LibC::HANDLE do
+    create_completion_port(LibC::INVALID_HANDLE_VALUE, nil)
+  end
+
+  def self.create_completion_port(handle : LibC::HANDLE, parent : LibC::HANDLE? = iocp)
+    iocp = LibC.CreateIoCompletionPort(handle, parent, nil, 0)
+    if iocp.null?
+      raise IO::Error.from_winerror("CreateIoCompletionPort")
+    end
+    iocp
+  end
+
+  def self.wait_completion(timeout = nil)
+    result = LibC.GetQueuedCompletionStatusEx(iocp, out io_entry, 1, out removed, timeout, false)
```

Comment:
I take it this is the stub you were talking about as this implementation only handles a single fiber :)

Reply:
Yes, I forgot to add a comment about that.